### ISSUE TITLE
К заголовку подраздела «Легкая атлетика» не применен формат шрифта «жирный»

### ISSUE THREAD

**Issue by OlgaKondratieva** [OlgaKondratieva/TestProjectOlga#6]
**Как воспроизвести**

1. Открыть главную страницу  https://sportmail.ru
2. В навигационной панели навести курсор на раздел «Другие»
3. Выпадет список подразделов.
4. Смотрим на "Легкая атлетика"

**Фактический результат**

К заголовку подраздела «Легкая атлетика» не применен формат шрифта «жирный». Остальные подразделы выделены.

![Снимок экрана 2020-07-26 в 16 14 _ шрифт текста](https://user-images.githubusercontent.com/54175989/88477736-029a2080-cf75-11ea-8048-aec9a83c9ae8.png)

**Ожидаемый результат**
К заголовкам всех подразделов применены одинаковые настройки формата шрифта.

**Comment by Svetlana678** [comment]
Да, ошибка воспроизводится, только у меня еще кроме раздела "Легкой атлетики" нежирным выделено "НБА".  
![image](https://user-images.githubusercontent.com/69030252/89447655-012fdb80-d75f-11ea-9b3a-c2d9383c7224.png)
По правилам оформления нужно было назначить на себя.

**Comment by OlgaKondratieva** [comment]
> Да, ошибка воспроизводится, только у меня еще кроме раздела "Легкой атлетики" нежирным выделено "НБА".
> ![image](https://user-images.githubusercontent.com/69030252/89447655-012fdb80-d75f-11ea-9b3a-c2d9383c7224.png)
> По правилам оформления нужно было назначить на себя.

Обратите внимание на мой скрин, на момент проведения теста, в списке не было "НБА", его добавили позже, соответственно ваше замечание не справедливо. Не внимательно сморите, а оценки снижаете.

**Comment by Svetlana678** [comment]
На момент проверки я написала,что увидела. Мои комментарии к Вашему багу потом будут проверять и решать, что нужно,а что нет и ставить оценки. Считайте это не замечанием, а комментарием, это в принципе так и называется. Я смотрела внимательно, поэтому я и написала, что ЕЩЕ КРОМЕ ЛЕГКОЙ АТЛЕТИКИ, смотрите внимательно.
